src/luro.rs

```diff
@@ -118,6 +118,7 @@ impl Luro {
                 commands: Luro::set_default_commands().into(),
                 guild_settings,
                 hecks,
+                interaction_count: Arc::from(tokio::sync::RwLock::new(0)),
```

Comment:
Likewise with my previous suggestion, no need to wrap this in an Arc.
```suggestion
                interaction_count: tokio::sync::RwLock::new(0),
```

Reply:
Done